js/services/Utils.js

```diff
@@ -0,0 +1,29 @@
+const Utils = {
+    // --------------------------------
+    //  Parse a url and break it into resource, id and verb
+    // --------------------------------
+    parseRequestURL : () => {
+
+        let url = location.hash.slice(1).toLowerCase() || '/';
+        let r = url.split("/")
+        let request = {
+            resource    : null,
+            id          : null,
+            verb        : null
+        }
+        request.resource    = r[1]
+        request.id          = r[2]
+        request.verb        = r[3]
+
+        return request
+    }
+
+    // --------------------------------
+    //  Simple sleep implementation
+    // --------------------------------
+    , sleep: (ms) => {
```

Comment:
Pourquoi tu fais un sleep ?

Reply:
ça sert pas trop ici. That's why j'ai mis `WIP` (Work In Progress)